gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -48,8 +48,8 @@ export default function NxFormLayoutExample() {
       <div className="nx-form-group">
         <label className="nx-label">
           <span className="nx-label__text">A Field to Fill in</span>
-          <NxStatefulTextInput aria-required={true} validator={validator}/>
         </label>
+        <NxStatefulTextInput aria-required={true} validator={validator}/>
```

Comment:
IIRC we were deliberately keeping some inconsistency in the examples given that we consider both ways to be currently supported.

Reply:
> IIRC we were deliberately keeping some inconsistency in the examples given that we consider both ways to be currently supported.

I think that we should settle on a "correct" way, and demonstrate only that. I'm not suggesting that we pull support for the other ways just that we maintain consistency in our examples. I created a ticket to go through and make the other example pages consistent.

Reply:
> I think that we should settle on a "correct" way, and demonstrate only that. I'm not suggesting that we pull support for the other ways just that we maintain consistency in our examples. I created a ticket to go through and make the other example pages consistent.

The problem is that we can't know for sure that we haven't broken a way that we don't have a demonstration of.

Reply:
> > I think that we should settle on a "correct" way, and demonstrate only that. I'm not suggesting that we pull support for the other ways just that we maintain consistency in our examples. I created a ticket to go through and make the other example pages consistent.
> 
> The problem is that we can't know for sure that we haven't broken a way that we don't have a demonstration of.

Well we could create a some kind of semi-hidden legacy page, but I don't think that we want to be stuck in a position where we endlessly support every historical variation of HTML layout. We should demonstrate what we consider to be the "correct" way of laying out the code. In this case I'm also resolving an inconsistency between the `fieldset` and the `form-group`.

Also I'm not removing any capabilities, I'm only making the examples consistent.

Reply:
> > I think that we should settle on a "correct" way, and demonstrate only that. I'm not suggesting that we pull support for the other ways just that we maintain consistency in our examples. I created a ticket to go through and make the other example pages consistent.
> 
> The problem is that we can't know for sure that we haven't broken a way that we don't have a demonstration of.

As discussed in WW I created this ticket to create deprecated example(s): https://issues.sonatype.org/browse/RSC-436